app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -43,6 +53,50 @@ class Identification extends React.Component {
     );
   };
 
+  // Special function for handling race selection: If an exclusive race value (e.g. "unknown" or "refused to answer")
+  // is selected, unselects all other checkboxes, and if a standard race value (e.g. "white, black or african american", "other")
+  // is selected, unselects all exclusive checkboxes.
+  handleRaceChange = event => {
+    let value = event.target.checked;
+    var modified_races = {};
+    let current = this.state.current;
+    let modified = this.state.modified;
+
+    const self = this;
+    event.persist();
+    if (value) {
+      if (self.exclusive_race_options.includes(event.target.id)) {
+        modified_races = _.reduce(
+          _.union(this.exclusive_race_options, this.default_races),
+          (memo, race) => {
+            return _.extend(memo, { [race]: race == event.target.id });
```

Comment:
A lot of lodash functions are used here (and below) that I think can just be replaced with using the spread operator. It's kind of hard to read as it is now.

Examples for a few of these lines:
Line 70:
```
{...this.exclusive_race_options, ...this.default_races}
```
Line 72:
```
return {...memo, { [race]: race == event.target.id };
```
Line 91:
```
current: { ...current, patient: { ..._.extend(current.patient, modified_races) } }
```

Reply:
this implementation was replaced and is no longer relevant: https://github.com/SaraAlert/SaraAlert/pull/689/commits/767b8765f1830cb0e34e8bc8b1adbbe48c0786f5#diff-0b38c12b7f12dd9f316a576d5b5dc6c12f0b8019720598550b239fca8828d292L77